segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerRecoveryUtils.java

```diff
@@ -246,6 +249,80 @@ private static void validateContainerIds(Map<Integer, DebugStreamSegmentContaine
                 ex -> ex instanceof StreamSegmentNotExistsException, null);
     }
 
+
+    /**
+     * Copies the contents of container metadata segment and its attribute segment to new segments. The new names to the segments
+     * are passed as parameters.
+     * @param storage                    A {@link Storage} instance where segments are stored.
+     * @param containerId                A Container Id to get the name of the metadata segment.
+     * @param backUpMetadataSegmentName  A name to the back up metadata segment.
+     * @param backUpAttributeSegmentName A name to the back attribute segment.
+     * @param executorService            A thread pool for execution.
+     * @return                           CompletableFuture which when completed will have the segments' contents copied to another segments.
+     */
+    public static CompletableFuture<Void> backUpMetadataAndAttributeSegments(Storage storage, int containerId,
+                                                                             String backUpMetadataSegmentName,
+                                                                             String backUpAttributeSegmentName,
+                                                                             ExecutorService executorService) {
+        Preconditions.checkNotNull(storage);
+        String metadataSegmentName = NameUtils.getMetadataSegmentName(containerId);
+        String attributeSegmentName = NameUtils.getAttributeSegmentName(metadataSegmentName);
+        return copySegment(storage, metadataSegmentName, backUpMetadataSegmentName, executorService)
+                        .thenAcceptAsync(x -> copySegment(storage, attributeSegmentName, backUpAttributeSegmentName,
+                                executorService));
+    }
+
+    /**
+     * Given the back up of metadata segments, this method iterates through all segments in it and gets their attributes
+     * and updates it in the container metadata segment of the given {@link DebugStreamSegmentContainer} instance.
+     * @param metadataSegments  A map of back of metadata segments along with their container Ids.
+     * @param containersMap     A map of {@link DebugStreamSegmentContainer} instances with their container Ids.
+     * @param executorService   A thread pool for execution.
+     * @throws Exception        If an exception occurred. This could be one of the following:
```

Comment:
Why not `@throws TimeoutException` and `@throws IOException`? That is the standard Javadoc we should use.

The format you used was our convention for async methods that return CompletableFuture.

Reply:
Done.